server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -0,0 +1,155 @@
+package com.google.musicanalysis.site;
+
+import com.google.gson.*;
+import com.google.musicanalysis.api.naturallanguage.*;
+import com.google.musicanalysis.api.perspective.*;
+import com.google.musicanalysis.api.youtube.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/analysis")
+public class AnalysisServlet extends HttpServlet {
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse res)
+      throws ServletException, IOException {
+
+    res.getWriter().write("<h1>Comment Analysis</h1>\n");
+
+    String videoName = "seun omonije";
+
+    String videoIdJson = new YoutubeRequest("search", videoName).getResult();
+    String videoId = getVideoId(videoIdJson);
+    String commentsJson = new YoutubeRequest("commentThreads", "snippet", videoId).getResult();
+
+    res.getWriter().write(commentsJson);
+
+    res.getWriter().write("<h3>Comments</h3>");
+    String comment = "I love you.";
+    res.getWriter().println(comment);
+
+    res.getWriter().write("<h3>Perspective</h3>");
+    HashMap<String, String> perspectiveMap = analyzeWithPerspective(comment);
+    // Print on the screen (won't make it in final cut)
+    Set set = perspectiveMap.entrySet();
+    Iterator mapIterator = set.iterator();
+    while (mapIterator.hasNext()) {
+      Map.Entry entry = (Map.Entry) mapIterator.next();
+      res.getWriter().println(entry.getKey() + ": " + entry.getValue());
+    }
+
+    res.getWriter().write("<h3>NLP</h3>");
+    MagnitudeAndScore nlpObject = analyzeWithNLP(comment);
+    res.getWriter().println("Magnitude: " + nlpObject.magnitude + "Score: " + nlpObject.score);
+  }
+
+  private class MagnitudeAndScore {
```

Comment:
Consider renaming this something like NLPResult. The current name isn't flexible for any future changes to this class.